exporter/exporterhelper/logs.go

```diff
@@ -27,14 +28,14 @@ var (
 type logsRequest struct {
 	ld         plog.Logs
 	pusher     consumer.ConsumeLogsFunc
-	cachedSize int
+	cachedSize map[exporterbatcher.SizerType]int
```

Comment:
Better to have 2 values than this map which needs heap allocation and lookup is more expensive.